.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,19 @@
-<!-- describe the changes you have made here: what, why, ... 
-     Link issues by using the following pattern: [#333](https://github.com/JabRef/jabref/issues/333) or [koppor#49](https://github.com/koppor/jabref/issues/47).
-     The title of the PR must not reference an issue, because GitHub does not support autolinking there. -->
+<!-- 
+Describe the changes you have made here: what, why, ... 
+Link issues by using the following pattern: #333.
+If you fixed a koppor issue, link it with following pattern: [koppor#47](https://github.com/koppor/jabref/issues/47).
+The title of the PR must not reference an issue, because GitHub does not support autolinking there.
+-->
 
+## Checklist
 
-----
+I went through the following checklist and checked all items. All items with `[ ]` are still a TODO. All items checked with `[x]` are done.
```

Comment:
This should be a comment, right?

Reply:
No, I wanted to it to be rendered as text so that the user really takes care about the check list.

Reply:
It's however inconsistent with the other approaches when creating issues and about the description of a PR.